langchain-tool-retrieval.ipynb

```diff
@@ -1,14 +1,37 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# OpenBB Agents"
+   ]
+  },
   {
    "cell_type": "code",
-   "execution_count": 22,
+   "execution_count": 34,
```

Comment:
It might be worth doing a "restart kernel" and "run all cells" from top to bottom, so that:
1. The outputs we show are accurate
2. We don't accidentally have stale state that will prevent the notebook from running top-to-bottom "as is".